doc/user_guide/io.rst

```diff
@@ -578,15 +578,17 @@ Extra loading arguments
 select_type: One of ('spectrum', 'image'). If specified just selected type of
 data is returned. (default None)
```

Comment:
Improvement suggestion: 
select_type: one of (None, 'spectrum', 'image'). If specified, only the corresponding type of data, either spectrum or image, is returned. By default (None), all data are loaded."